components/Statistics/StatisticsActivityTagsReport.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import { LoadSpinner } from '../Loading'
+import { PieChart, Pie, Tooltip, Legend, Cell, ResponsiveContainer } from 'recharts'
+import styled from 'styled-components'
+import { useStatisticsAPI } from '../../lib/statistics/statisticsHooks'
+
+const COLORS = ['#0088FE', '#00C49F', '#FFBB28', '#FF8042', '#653CAD']
+
+const Container = styled.div`
```

Comment:
Although its internal to this module Container is rather a generic element name.  Also this set of options seems likely to have been defined already - check the existing styled components.

Also consider using a semantic component rather than a div.  is this container an <article> or a <section> for example?